R/Gargle-class.R

```diff
@@ -0,0 +1,213 @@
+#' Generate a Gargle token
+#'
+#' Constructor function for objects of class [`Gargle2.0`]. The `"email"` scope
```

Comment:
I think this should be `` `[Gargle2.0]` ``

Reply:
OK I tried that but it breaks the links. I'm doing what it says in the roxygen md vignette, which, as you know, I visit frequently!

https://github.com/klutometis/roxygen/blame/e2ed231a17644fb85b73a1d31bb1c84cb3bb7893/vignettes/markdown.Rmd#L149-L152

Reply:
Although that bullet point seems to contradict itself 🤔

Reply:
Oh, it should just be `[Gargle2.0]`

Reply:
ok